src/main/java/uk/gov/companieshouse/accounts/association/controller/AssociationsListForCompanyController.java

```diff
@@ -27,25 +27,26 @@ public AssociationsListForCompanyController(AssociationsService associationsServ
 
     @Override
     public ResponseEntity<AssociationsList> getAssociationsForCompany( final String companyNumber, final String xRequestId, final Boolean includeRemoved, final Integer pageIndex, final Integer itemsPerPage) {
-
-        LOG.debugContext(xRequestId, String.format( "Attempting to fetch users that are associated with company %s. includeRemoved=%b, itemsPerPage=%d, and pageIndex=%d.", companyNumber, includeRemoved, itemsPerPage, pageIndex ),null );
+        LOG.infoContext( xRequestId, "Routing request to GET /associations/companies/{company_number}.", null );
```

Comment:
hey this requests logging should be handled by interceptors. rather than adding them manually.
you can configure interceptors. see common logger library we use.

Reply:
Good spot - you are right. This information is already being logged by the interceptors, so it is redundant. I will remove the redundant logging statement.